app/constants.js

```diff
@@ -29,23 +28,23 @@ export default {
 
   // documentation URLs
   DOC_URLS: {
-    GENERAL: '/_utils/docs/intro/api.html#documents',
-    ALL_DBS: '/_utils/docs/api/server/common.html?highlight=all_dbs#get--_all_dbs',
-    REPLICATION: '/_utils/docs/replication/replicator.html#basics',
-    DESIGN_DOCS: '/_utils/docs/ddocs/ddocs.html#design-docs',
-    DESIGN_DOC_METADATA: '/_utils/docs/api/ddoc/common.html#api-ddoc-view-index-info',
-    VIEW_FUNCS: '/_utils/docs/ddocs/ddocs.html#view-functions',
-    MAP_FUNCS: '/_utils/docs/ddocs/ddocs.html#map-functions',
-    REDUCE_FUNCS: '/_utils/docs/ddocs/ddocs.html#reduce-and-rereduce-functions',
-    API_REF: '/_utils/docs/http-api.html',
-    DB_PERMISSION: '/_utils/docs/api/database/security.html#db-security',
-    STATS: '/_utils/docs/api/server/common.html?highlight=stats#get--_stats',
-    ACTIVE_TASKS: '/_utils/docs/api/server/common.html?highlight=stats#active-tasks',
-    LOG: '/_utils/docs/api/server/common.html?highlight=stats#log',
-    CONFIG: '/_utils/docs/config/index.html',
-    VIEWS: '/_utils/docs/intro/overview.html#views',
-    MANGO_INDEX: '/_utils/docs/intro/api.html#documents',
-    MANGO_SEARCH: '/_utils/docs/intro/api.html#documents',
-    CHANGES: '/_utils/docs/api/database/changes.html?highlight=changes#post--db-_changes'
+    GENERAL:'./docs/intro/api.html#documents',
```

Comment:
I should probably use the getServerUrl here to be more consistent but I had trouble with circular dependencies 😞